README.md

```diff
@@ -41,7 +41,7 @@ In the `gunicorn` section, you can configure:
 
 ### Start the app
 
-    docker-compose up
+    CONFIG_FILE_PATH=/path/to/config.yaml docker-compose up
```

Comment:
I might clarify what kind of path should be used here: the path to the file on the host machine? A path within in the container?

Reply:
Within the container. That environment variable is used by `aumbry`.